src/app/[locale]/(default)/events/page.tsx

```diff
@@ -21,5 +26,84 @@ export default async function EventsPage({
 }) {
   const { locale } = await params;
   setRequestLocale(locale);
-  return <div>This should be events page</div>;
+  const t = await getTranslations('events');
+  const tUi = await getTranslations('ui');
+
+  const translations = {
+    internal: tUi('internal'),
+    startsAt: t('startsAt'),
+    startedAt: t('startedAt'),
+    endsAt: t('endsAt'),
+    endedAt: t('endedAt'),
+  };
+
+  return (
+    <>
+      <h1 className='my-4'>{t('title')}</h1>
+      <Link href='#active'>
+        <h2 className='my-2' id='active'>
+          {t('activeEvents')}
+        </h2>
+      </Link>
+      {events.slice(0, 1).map((event) => (
+        <EventCard
+          key={event.id}
+          wrapperClassName='block'
+          event={event}
+          t={{
+            detailsAboutEvent: t('detailsAboutEvent', {
+              eventName: event.title,
+            }),
+            photoOf: tUi('photoOf', { name: event.title }),
+            ...translations,
+          }}
+          _active
+        />
+      ))}
+      <Link href='#upcoming'>
+        <h2 className='my-4' id='upcoming'>
+          {t('upcomingEvents')}
+        </h2>
+      </Link>
+      <div className='grid grid-cols-1 gap-2 lg:grid-cols-2'>
+        {events.slice(1, 5).map((event) => (
+          <EventCard
+            key={event.id}
+            event={event}
+            wrapperClassName='lg:last:odd:col-span-2'
+            cardClassName='h-full'
+            t={{
+              detailsAboutEvent: t('detailsAboutEvent', {
+                eventName: event.title,
+              }),
+              photoOf: tUi('photoOf', { name: event.title }),
+              ...translations,
+            }}
+          />
+        ))}
+      </div>
+      <Link href='#past'>
+        <h2 className='my-4' id='past'>
```

Comment:
```suggestion
        <h2 className='pt-4 mb-4' id='past'>
```
mt-4 and my-4 doesn't add space to the top for some reason, pt-4 mb-4 will give the same styling as in loading.tsx